h2/src/test/scala/org/scalarelational/h2/modular/ModularSpec.scala

```diff
@@ -32,57 +32,57 @@ class ModularSpec extends WordSpec with Matchers {
       }
     }
     "connect handlers" in {
-      users.handlers.inserting.on {
+      users.handlers.inserting.attach {
```

Comment:
I think `on` is clearer what the purpose is than `attach` and is much more consistent with other event systems. Not a big deal really, but I presume you'd want the syntax as clear as possible.